lib/unleash/feature_toggle.rb

```diff
@@ -48,11 +48,14 @@ def get_variant(context, fallback_variant = Unleash::FeatureToggle.disabled_vari
 
       choice = evaluation_result.enabled? ? :yes : :no
       Unleash.toggle_metrics.increment_variant(self.name, choice, variant.name) unless Unleash.configuration.disable_metrics
+
+      variant.feature_enabled = evaluation_result.enabled? || false
```

Comment:
not sure if the `|| false` is needed as `is_enabled?`/`enabled?` methods should really always return a `bool`. (and if it doesn't, that's a pretty big bug somewhere).

Reply:
Thanks, TIL. Addressed in: https://github.com/Unleash/unleash-client-ruby/pull/197/commits/48e4f77cce8aba817833bb34391157f07511026f